src/models/configuration.js

```diff
@@ -94,6 +94,12 @@ export const schema = z.object({
   foreEdgePaddingPt: urlSafe(z.coerce.number()).default(0), // specific to wacky small
   wackySpacing, // specific to wacky small
   flyleafs: urlSafe(z.coerce.number()).default(1),
+
+  sewingMarksEnabled: urlSafe(coercedBoolean).default(false),
+  sewingMarksMarginPt: urlSafe(z.coerce.number()).default(72),
+  sewingMarksAmount: urlSafe(z.coerce.number()).default(3),
+  sewingMarksTapeWidthPt: urlSafe(z.coerce.number()).default(36)
```

Comment:
This line needs a comma at the end, that's why the build and preview are failing

Reply:
mistake during merge conflicts :).